homeassistant/components/freebox/config_flow.py

```diff
@@ -82,27 +73,35 @@ async def async_step_link(self, user_input=None) -> FlowResult:
             await fbx.close()
 
             return self.async_create_entry(
-                title=self._host,
-                data={CONF_HOST: self._host, CONF_PORT: self._port},
+                title=self._data[CONF_HOST],
+                data={
+                    CONF_HOST: self._data[CONF_HOST],
+                    CONF_PORT: self._data[CONF_PORT],
+                },
```

Comment:
```suggestion
                data=self._data,
```

The `_data` dictionary only holds the host and port